pkg/alexa/l10n/l10n.go

```diff
@@ -0,0 +1,246 @@
+package l10n
+
+import (
+	"fmt"
+	"github.com/drpsychick/alexa-go-cloudformation-demo/pkg/alexa"
+	"math/rand"
+	"time"
+)
+
+// Default keys
+const (
+	KeySkillName                Key = "SKILL_Name"
+	KeySkillDescription         Key = "SKILL_Description"
+	KeySkillSummary             Key = "SKILL_Summary"
+	KeySkillExamplePhrases      Key = "SKILL_ExamplePhrases"
+	KeySkillKeywords            Key = "SKILL_Keywords"
+	KeySkillSmallIconURI        Key = "SKILL_SmallIconURI"
+	KeySkillLargeIconURI        Key = "SKILL_LargeIconURI"
+	KeySkillTestingInstructions Key = "SKILL_TestingInstructions"
+	KeySkillInvocation          Key = "SKILL_Invocation"
+	KeySkillPrivacyPolicyURL    Key = "SKILL_PrivacyPolicyURL"
+	KeySkillTermsOfUse          Key = "SKILL_TermsOfUse"
+)
+
+func init() {
+	rand.Seed(time.Now().UTC().UnixNano())
+}
+
+func Speak(text string) string {
```

Comment:
This and others like it belong in an `ssml` package. They have nothing to do with translation.

Reply:
https://github.com/DrPsychick/alexa-go-cloudformation-demo/issues/29